nvm.sh

```diff
@@ -117,11 +117,17 @@ nvm_get_latest() {
 
 nvm_download() {
   local CURL_COMPRESSED_FLAG
+  local AUTH_HEADER_FLAG
+
+  if [ -n "$NVM_AUTH_HEADER" ]; then
+    AUTH_HEADER_FLAG="--header $NVM_AUTH_HEADER"
+  fi
+
   if nvm_has "curl"; then
     if nvm_curl_use_compression; then
       CURL_COMPRESSED_FLAG="--compressed"
     fi
-    curl --fail ${CURL_COMPRESSED_FLAG:-} -q "$@"
+    curl --fail ${CURL_COMPRESSED_FLAG:-} ${AUTH_HEADER_FLAG:-} -q "$@"
```

Comment:
```suggestion
    curl --fail ${CURL_COMPRESSED_FLAG:-}${AUTH_HEADER_FLAG:-} -q "$@"
```
that way the space is in the variable above ¯\\\_(ツ)\_/¯